spec/services/articles/creator_spec.rb

```diff
@@ -67,7 +67,7 @@
 
       expect(article.decorated?).to be(false)
       expect(article).not_to be_persisted
-      expect(article.errors.size).to eq(1)
+      expect(article.errors.size.positive?).to be(true)
```

Comment:
Why the change away from `eq(1)` here?

Reply:
Because with `nil` body and `in:` you get both the min and the max errors

Reply:
Right, so if only one of those validations fails here, this test still passes. Was that intentional?

Reply:
I reverted it now that I've moved the check to `bytesize` and a separate validator